component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -170,7 +170,7 @@ public static String getProfileLink(final String username, final String portalOw
                 .append(buildProfileUri(identity.getRemoteId(), null, portalOwner)).append("\" target=\"_parent\">")
                 .append(StringEscapeUtils.escapeHtml(identity.getProfile().getFullName()));
     if(identity.getProfile().getProperty("external") != null && identity.getProfile().getProperty("external").equals("true")){
-      profileLink = profileLink.append("<span \" class=\"externalTagClass\">").append(" (").append(getResourceBundleLabel(new Locale(lang), "external.label.tag")).append(")").append("</span>");
+      profileLink = profileLink.append("<span \" class=\"externalFlagClass\" style=\"color:#999999\">").append(" (").append(getResourceBundleLabel(new Locale(lang), "external.label.tag")).append(")").append("</span>");
```

Comment:
Please add the style to the CSS class, we should not add styles directly on HTML code.

Reply:
i tried many solutions but it seems that the vue component is not recognizing the class "externalTagClass" or "externalflagclass" because it's injected from the java backend code as belongign to  the entire mention message.

Reply:
We need to have a better way to fix this, no style to be put directly in the DOM

Reply:
@nidhalnidhal  you should force style color in the CSS class